setup/jest.config.js

```diff
@@ -274,8 +274,6 @@ module.exports = {
     'app/src/main.js',
```

Comment:
NRTTL:
Do we have an issue for updating the table on http://0.0.0.0:8080/#/wallet with this socket data as well? Now we can have a state where table is empty but we get a toast that we have received a transaction.

Reply:
Fixed in https://github.com/LiskHQ/lisk-desktop/pull/4626/commits/3b0ac529ba07dd2482afda70edb985786e4b0974